app/src/androidTest/java/org/simple/clinic/patient/PatientRepositoryAndroidTest.kt

```diff
@@ -3980,4 +3980,78 @@ class PatientRepositoryAndroidTest {
         patientWithAnotherFacilityAsAssignedFacility, patientWithCurrentFacilityAsAssignedFacility
     )
   }
+
+  @Test
+  fun querying_all_patients_in_facility_must_return_list_of_patients_partitioned_by_assigned_facility_and_order_by_name() {
+    fun createPatientWithNameAndAssignedFacilityID(
+        patientUuid: UUID,
+        patientName: String,
+        assignedFacilityId: UUID?
+    ) {
+      val patientProfile = testData
+          .patientProfile(patientUuid = patientUuid, patientName = patientName, patientAssignedFacilityId = assignedFacilityId)
+          .let { patientProfile ->
+            patientProfile.copy(
+                patient = patientProfile.patient.copy(fullName = patientName, assignedFacilityId = assignedFacilityId),
+            )
```

Comment:
Isn't this copy method redundant? We are already setting these properties when constructing the patient profile.

Reply:
Ah right, I copied the above implementation since it's mostly the same. Will fix it 👍🏽